misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -912,7 +914,7 @@ public SimpleNameReferenceNode transform(SimpleNameReferenceNode simpleNameRefer
 
     @Override
     public IfElseStatementNode transform(IfElseStatementNode ifElseStatementNode) {
-        if (!isInLineRange(ifElseStatementNode, lineRange)) {
+        if (!isInLineRange(ifElseStatementNode, lineRange) || !nestedIfBlock(ifElseStatementNode).isEmpty()) {
```

Comment:
This will create an impact when formatting the following type of nested-if code. 

```
public function foo() {
    if (true) {
        
        if (1 < 2) {
            int x = 2;
        }
    } else {
        int y = 0;
        return i;
    }
}
```

Reply:
Yeah, that is added to disable the formatting of nested if blocks, since this fix requires some time. The intention is to leave the nested if block as it is now. The following issue is there to track the fix.
https://github.com/ballerina-platform/ballerina-lang/issues/25999

Reply:
Since the formatting of nested if blocks are disabled for now if there are formatting errors in the above code snippet, it will not get formatted. 

But if the above code is formatted as it is, then the output will also be the same.